.github/workflows/validate-pull-request.yaml

```diff
@@ -5,6 +5,7 @@ on:
       - main
 jobs:
   pr-validation:
+    name: PR Validation
```

Comment:
maybe change the name of this workflow (and file) to indicate this only happens for main.